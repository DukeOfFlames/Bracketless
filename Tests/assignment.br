START
째a = "Hi (with Quotes)"
째b = a
째c = {째d = 1}
#drucke{a}
#drucke{b}
#drucke{c}
#drucke{d}
END