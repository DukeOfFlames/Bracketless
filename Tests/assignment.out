"Hi (with Double Quotes)"
"Hi (with Double Quotes)"
"Hi (with Single Quotes)"
"Hi (with Single Quotes)"
1
1
